plugins/plugin-hosted/codenvy-hosted-ext-factory/src/main/java/com/codenvy/ide/factory/client/utils/FactoryProjectImporter.java

```diff
@@ -132,6 +132,10 @@ public void apply(List<ProjectConfigDto> projectConfigs) throws OperationExcepti
                 Set<String> projectNames = new HashSet<>();
                 String createPolicy = factory.getPolicies() != null ? factory.getPolicies().getCreate() : null;
                 for (ProjectConfigDto projectConfig : projectConfigs) {
+                    if (projectConfig.getSource().getLocation() == null) {
```

Comment:
Yes, sure (i thought it was) This https://github.com/eclipse/che/pull/1056/files#diff-fc34b5aa12e2c9b6e3b005a8987613ccR603 line confused me